noxfile.py

```diff
@@ -45,6 +59,26 @@ def lint(session):
     )
 
 
+@nox.session(python="3.8")
+def format(session):
+    """
+    Run isort to sort imports. Then run black
+    to format code to uniform standard.
+    """
+    session.install(BLACK_VERSION, ISORT_VERSION)
+    # Use the --fss option to sort imports using strict alphabetical order.
+    # See https://pycqa.github.io/isort/docs/configuration/options.html#force-sort-within-sections
+    session.run(
```

Comment:
Aren't we usually using `--profile google` to sort? IIRC linter was complaining when I tried using isort with default profile.